en/airframes/adding_a_new_frame.md

```diff
@@ -115,12 +115,14 @@ S: 0 1   6500   6500      0 -10000  10000
 
 Where each number from left to right means:
 
-* M: Indicates two scalers for two inputs
-* O: Indicates the output scaling (*1 in negative, *1 in positive), offset (zero here), and output range (-1..+1 here).  If you want to invert your PWM signal, the signs of the output scalings has to be changed. (```O:      -10000  -10000      0 -10000  10000```)
-* S: Indicates the first input scaler: It takes input from control group #0 (attitude controls) and the first input (roll). It scales the input * 0.6 and reverts the sign (-0.6 becomes -6000 in scaled units). It applies no offset (0) and outputs to the full range (-1..+1)
-* S: Indicates the second input scaler: It takes input from control group #0 (attitude controls) and the second input (pitch). It scales the input * 0.65 and reverts the sign (-0.65 becomes -6500 in scaled units). It applies no offset (0) and outputs to the full range (-1..+1)
+* M: Indicates two scalers for two control inputs. It indicates the number of control inputs mixer will receive.
+* O: Indicates the output scaling (*1 in negative, *1 in positive), offset (zero here), and output range (-1..+1 here).  If you want to invert your PWM signal, the signs of the output scalings have to be changed. (```O:      -10000  -10000      0 -10000  10000```)
```

Comment:
- what does "(*1 in negative, *1 in positive)" mean? - multiplying by one doesn't do anything, and these appear to be same. 
- output range is marked as "output range (-1..+1 here)" but the output value is listed as "-10000  10000" - ie the numbers aren't the same

So the descriptions don't make much sense. It would be good to understand what the output scaling, range and offset actually do - ie what ranges make sense.